app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,16 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // force interger input and submit on enter
+      var x = event.which;
+      if ((x > 8 && x < 48) || (x > 57 && x < 96) || x > 105) {
+        event.preventDefault();
+      } else if (x === KeyCodes.ENTER) {
```

Comment:
You left the `submitOnEnter` on enter function in this file, is it still needed?